src/Cli.Tests/UpdateEntityTests.cs

```diff
@@ -1093,7 +1093,9 @@ private static UpdateOptions GenerateBaseUpdateOptions(
             IEnumerable<string>? relationshipFields = null,
             IEnumerable<string>? map = null,
             IEnumerable<string>? restMethodsForStoredProcedure = null,
-            string? graphQLOperationForStoredProcedure = null
+            string? graphQLOperationForStoredProcedure = null,
+            string? cacheEnabled = null,
```

Comment:
We should add automated tests that test if the these cache option get updated or not using the flags in the update command.